torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -237,6 +241,12 @@ bool IsValidONNXNode(const Node* n) {
     }
   }
 
+  for (auto inp : n->inputs()) {
+    if (inp->type() == NoneType::get()) {
+      return false;
```

Comment:
Found and fixed the only known case in `FixupONNXSubblockOutputs` by using empty `onnx::Optional` instead of `onnx::Identity(None)`. Will address other cases as needed.